pkg/restore/restore_test.go

```diff
@@ -2410,88 +2407,6 @@ func TestRestoreWithRestic(t *testing.T) {
 	}
 }
 
```

Comment:
I'd add a TODO here that we add a new test for the `getOrderedResources` function.

Reply:
:+1:

Reply:
I added a test.